src/Runtime/Distributions/DiscreteChar.cs

```diff
@@ -199,14 +199,14 @@ private Storage Data
         /// </summary>
         public char Point
         {
-            get => this.Data.Point;
+            get => this.Data.Point ?? throw new InvalidOperationException();
```

Comment:
For distributions in general, Point is supposed to always succeed even if the distribution is not a point mass. So, in general it might return the mode or the mean. This is possibly the only distribution for which this does not hold. I realize that this current change has not given rise to this, but it may be worthwhile tracking back through your previous changes to see if this was originally the case.

Reply:
It used to throw an exception before my changes. I can recreate proposed change of behaviour in separate PR.